app/modules/sr/index.js

```diff
@@ -44,6 +45,14 @@ export default angular.module('xoWebApp.sr', [
 
     let {get} = xoApi
     $scope.$watch(() => xoApi.get($stateParams.id), function (SR) {
+      if (SR) {
+        forEach(SR.VDIs, vdi => {
+          vdi = xoApi.get(vdi)
+          if (vdi && isNumber(vdi.size)) {
```

Comment:
If already a string, `bytesToSize()` could be updated to return directly its argument.